app/src/org/commcare/utils/FormUploadResult.java

```diff
@@ -10,38 +10,51 @@
     FULL_SUCCESS(0),
 
     /**
-     * There was a problem with the server's response
+     * Non-specific error used as default assumption before attempting submission, and when
+     * certain exceptions are encountered (IOException or TaskCancelledException)
      */
     FAILURE(1),
 
     /**
-     * The server returned an authentication error
+     * There is a problem with this record that prevented submission success for this record
+     * specifically, but others will be allowed to continue trying
      */
-    AUTH_FAILURE(2),
+    RECORD_FAILURE(2),
```

Comment:
@ctsims I reordered these a little bit because the behavior of `FormUploadResult.getWorstResult` implies that the worse a result is, the higher the `orderVal` should be. The ordering that previously existed here did not, in my understanding of things, accurately reflect that, but I'm not sure if the one I settled on is correct either. Would appreciate your input, to the extent that you think it's important to get this right.